proxy/logging/LogAccessTest.cc

```diff
@@ -300,6 +300,19 @@ LogAccessTest::marshal_proxy_req_header_len(char *buf)
   -------------------------------------------------------------------------*/
 
 int
+LogAccessTest::marshal_proxy_req_net_hdr_len(char *buf)
```

Comment:
ACAICT ``LogAccessTest`` is never actually used. It would be great if you could look into getting this stuff into unit tests (probably in ``traffic_server`` regressions).